CMakeLists.txt

```
@@ -66,61 +67,62 @@ if(DEFINED QT_VERSION)
   set(CMAKE_AUTOUIC ON)
 
   set(
-      VISUALIZER_CORE_SOURCES
-      ${CMAKE_SOURCE_DIR}/src/visualizer/settings/SettingsRegistry.cpp
+          VISUALIZER_CORE_SOURCES
+          ${CMAKE_SOURCE_DIR}/src/visualizer/settings/SettingsRegistry.cpp
```

Same comment from l:39-l:41
```suggestion
     VISUALIZER_CORE_SOURCES
     ${CMAKE_SOURCE_DIR}/src/visualizer/settings/SettingsRegistry.cpp
```